lnrpc/lightning.proto

```diff
@@ -2123,11 +2123,81 @@ message BatchOpenChannel {
     */
     bytes pending_chan_id = 8;
 
+    /*
+    The maximum amount of coins in millisatoshi that can be pending within
+    the channel. It only applies to the remote party.
+    */
+    uint64 remote_max_value_in_flight_msat = 9;
+
+    /*
+    The maximum number of concurrent HTLCs we will allow the remote party to add
+    to the commitment transaction.
+    */
+    uint32 remote_max_htlcs = 10;
+
+    /*
+    Max local csv is the maximum csv delay we will allow for our own commitment
+    transaction.
+    */
+    uint32 max_local_csv = 11;
+
     /*
     The explicit commitment type to use. Note this field will only be used if
     the remote peer supports explicit channel negotiation.
     */
-    CommitmentType commitment_type = 9;
+    CommitmentType commitment_type = 12;
```

Comment:
We can't just change the type numbers of existing fields! We need to add the new fields afterwards, otherwise we break backward compatibility.